src/Notify/Notify.csproj

```diff
@@ -16,7 +16,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="JWT" Version="[6.0,7.3]" />
+    <PackageReference Include="JWT" Version="[7.0, 8.0)" />
```

Comment:
Is it at all of note that previously `6.0,7.3` has no space between it and now `7.0, 8.0` does?

Reply:
The tests seemed to work with the space, but I've removed it just to be safe because [the docs](https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges) don't include a space. And also because the tests didn't actually pick up that the last version didn't work...